buildengine/testdata/projects/highgoversion/go.mod

```diff
@@ -42,5 +42,3 @@ require (
 	golang.org/x/text v0.14.0 // indirect
 	google.golang.org/protobuf v1.33.0 // indirect
 )
-
-replace github.com/TBD54566975/ftl => /Users/dli/Development/ftl
```

Comment:
@deniseli hopefully this doesn't break anything you were testing before, but assuming we don't want your local path here. Lemme know if you want me to change this back or to something else.

Reply:
This should be turned into a relative replace path.

Reply:
Word! Will update.